gcn/notices/icecube/LvkNuTrackSearch.example.json

```diff
@@ -1,43 +1,50 @@
 {
   "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/icecube/LvkNuTrackSearch.schema.json",
   "type": "IceCube LVK Alert Nu Track Search",
-  "reference": { "gcn.notices.LVK.alert": 4642 },
-  "ref_ID": "MS230427r",
-  "alert_datetime": "2023-04-16T05:32:29.55Z",
-  "trigger_time": "2023-04-16T05:22:26.0Z",
-  "observation_start": "2023-04-16T05:14:06.0Z",
-  "observation_stop": "2023-04-16T05:30:46.0Z",
+  "reference": { "gcn.notices.LVK.alert": "S230914ak" },
+  "ref_ID": "S230914ak-2-Preliminary",
```

Comment:
@blaufuss Is it ok to swap as following:
"ref_ID": { "gcn.notices.LVK.alert": "S230914ak" },
"reference": "S230914ak-2-Preliminary",

Reply:
we can make that work, if that's the desired format from you guys, but it's not what I'm requesting...

Reply:
just to be clear it's a much larger change to the base schema to do that...

Reply:
sorry edit is :"ref_ID": "S230914ak" ,
"reference": "S230914ak-2-Preliminary",

Reply:
we have yet to include  the "reference" properly in our alerts, that's fine with us, but I don't know about other notice producers if you change from a dict to a string...

Reply:
Example as follows, ref id is trigger number and reference is specific lvk notice
"ref_ID": "S230914ak" ,
"reference": {"gcn.notices.LVK.alert": "S230914ak-2-preliminary"} ,

We are processing reference as union, both number or string are allowed (https://github.com/nasa-gcn/gcn-schema/pull/107). Let me know your views.

Reply:
This fine for us, will update our PR for an our updated example ASAP